package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@comunica/actor-init-sparql": "^1.5.0",
     "@comunica/actor-rdf-resolve-quad-pattern-rdfjs-source": "^1.5.4",
+    "@comunica/actor-http-solid-auth-fetch": "^1.0.0",
     "mkdirp": "^0.5.1"
```

Comment:
Not caused by this PR, but I assume `mkdirp` should be a dev dependency.

Reply:
Will fix.